### PR TITLE
DCOS_OSS-2065: accept floating points in FrameworkConfigurationForm

### PR DESCRIPTION
**Pull request by mperrotti**
## Testing
Use a browser other than Chrome.. With your browser and OS in locale that uses "," as a decimal separator  (de-DE, ru, fr-FR, etc...).
1. Go to Catalog and add a new service
2. Go to the "Nodes" tab
3. Change CPUs to another floating point

The field should not be considered invalid

## Trade-offs
- Setting `novalidate` on this form makes us miss out on browser validation (e.g.: people can put letters into an `input[type="number"]`)
- We're setting `novalidate` on this form, but there may be other forms where we have this issue that we just haven't caught yet

## Dependencies
None

## Screenshots
Before:
floatingpoint-broken

After:
floatingpoint-fixed

### CODE REVIEW

**Comment by nLight** [comment]
I bet this needs to be done https://wiki.mesosphere.com/display/ENGINEERING/Extract+and+update+translation+messages

File: locale/zh/messages.json [hunk 1]
```diff
@@ -12331,6 +12340,15 @@
       ]
     ]
   },
+  "seconds": {
+    "translation": "",
+    "origin": [
+      [
+        "plugins/services/src/js/containers/service-debug/TaskStatsTable.js",
```
Comment (nLight) [inline locale/zh/messages.json]: 🤔 rebase?

**Comment by mperrotti** [comment]
@nLight - I tried updating langpacks, but got no changes 🤔

**Comment by nLight** [comment]
Oh, I know why! That file doesn't have _any_ translations does it?

**Comment by mperrotti** [comment]
No, it appears to have translations...
Here are two:
https://github.com/dcos/dcos-ui/blob/867bf2f6f1471c4ec00938e508d5ff553d9b4338/src/js/components/FrameworkConfigurationForm.js#L103

https://github.com/dcos/dcos-ui/blob/867bf2f6f1471c4ec00938e508d5ff553d9b4338/src/js/components/FrameworkConfigurationForm.js#L338

**Comment by GeorgiSTodorov** [comment]
After thinking about this and https://jira.mesosphere.com/browse/DCOS-45866 I think that adding some form of validation if not in this PR then at least in another jira will be a very good idea.

**Comment by mperrotti** [comment]
@GeorgiSTodorov - we already have our own validation:
screen shot 2018-12-14 at 3 35 31 pm

`no-validate` is just telling the browser not to do it's own validation

And the value that gets sent is formatted with a period - you can see this by changing the number (while using a comma separated decimal) and looking at the `value` attribute of the `<input />`

**Comment by AmrAbdelrazik** [comment]
@mperrotti  and @GeorgiSTodorov ,  to @jaxesn 's point, I thought we inherit the validation from the jsonschema, why do we have to do anything the form code for this specific field?

**Comment by GeorgiSTodorov** [comment]
@AmrAbdelrazik , I am not that familiar with jsonschema, but I think it was HTML5 validation causing this.

**Comment by AmrAbdelrazik** [comment]
Please ping @nLight for more information, all form fields in DC/OS catalog are auto generated from a JSON schema object that specifies the field and the required validation. I want to understand why this exact field is behaving differently from that behavior.

**Pull request merged**